docs/guides/runtime-options-overview.mdx

```diff
@@ -18,11 +19,294 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
+For V1 primitives, if you do not specify a value for an option, the Qiskit Runtime client's default value is used. In V1, the default value is dependant on the code version.
+
+The tables in the [Options classes summary](#options-classes) section lists the default values.
+
+<span id="options-precedence"></span>
+### Precedence rules
+
+There are several ways to set options.  If you set different values for the same option in multiple places, this is the order of precedence:
+
+1. Any options set after primitive initialization.
+2. Any options set during primitive initialization.
+
+<Admonition type="caution" title="Important">
+Shots (Sampler) or precision (Estimator) set in `run()` are not considered options, but take precidence over values set for `default_shots`or `default_precision`. See the [Run() method](#run-method) section for full details.
+</Admonition>
+
+In the following example, these values would be used for the `estimator.run` job: `shots = 128` and `optimization_level = 1`.
+
+``` python
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={"optimization_level": 0})
+
+# Setting options after primitive initialization
+# This uses auto complete.
+estimator.options.optimization_level = 1
+estimator.options.default_shots = 4000
+
+# Sample two circuits at 128 shots each.
+estimator.run([circuit1, circuit2], shots=128)
+```
+
+#### Special cases
+
+The `default_shots` and `resilience_level` options require special consideration:
+
+If you set `default_shots` in an estimator, it overrides any value set for `default_precision`.
+
+The `resilience_level` is used to set a base configuration on any relevant options, and then any other user-specified options override that configuration. So it would be possible to undo the effect of setting the resilience level by reversing all of its settings manually.
+
+In the following example, setting the resilience level to 0 does not turn off `zne_mitigation` because `estimator.options.resilience.zne_mitigation = True` overrides the relevent setup from `estimator.options.resilience_level = 0`.
+
+``` python
+from qiskit_ibm_runtime import EstimatorV2, QiskitRuntimeService
+from qiskit import QuantumCircuit
+
+service = QiskitRuntimeService()
+backend = service.backend("ibm_auckland")
+
+estimator = EstimatorV2(backend)
+
+estimator.options.default_shots = 100
+estimator.options.resilience_level = 0
+estimator.options.resilience.zne_mitigation = True
+```
+
+<span id="options-classes"></span>
+## Options classes summary
+
+<Tabs>
+  <TabItem value="EstimatorV2" label="Estimator V2">
+  - [Resilience](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2): Advanced options for configuring error mitigation methods such as measurement error mitigation, ZNE, and PEC. Estimator only.
```

Comment:
It doesn't really need `Estimator only` since it's already listed under EstimatorV2 tab